backend/utils/deployShop.js

```diff
@@ -228,6 +297,9 @@ async function deployShop({
   if (subdomain) {
     domain = dnsProvider ? `https://${subdomain}.${zone}` : null
     await configureShopDNS({ network, subdomain, zone, hash, dnsProvider })
+    if (network.ipfs) {
+      triggerAutoSSL(domain, network.ipfs)
```

Comment:
I assume we intentionally want to run this in the background so no await.
Perhaps add a comment explaining this.

Reply:
Will add a note